dynacred/src/Storage.ts

```diff
@@ -32,13 +32,22 @@ export class StorageDB {
     }
 
     static async getObject(entry: string): Promise<any> {
-        const obj = JSON.parse(await this.get(entry), (key, value) => {
-            if (value && typeof value === "object" && value.type === "Buffer") {
-                return Buffer.from(value);
-            }
-            return value;
-        });
-        return obj == null ? {} : obj;
+        const d = await this.get(entry);
+        if (d === "") {
+            return {};
+        }
+        try {
+            const obj = JSON.parse(d, (key, value) => {
+                if (value && typeof value === "object" && value.type === "Buffer") {
+                    return Buffer.from(value);
+                }
+                return value;
+            });
+            return obj == null ? {} : obj;
```

Comment:
From the  [documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/parse), `JSON.parse` can't return null.